docs/src/main/sphinx/overview/concepts.rst

```diff
@@ -27,11 +27,35 @@ general to most specific.
     provide further information about Trino and the concepts in use.
 
 
-Server types
+.. _trino-concept-architecture:
```

Comment:
Is there an advantage to adding an anchor if we're not going to use it?

I think there's a few others like this.

Reply:
Maybe it's preemptive but I think making the standard that we add these when defining glossary entries or concepts would be good, that way it's not a guessing game of if a term has a link already set up. Plus, doing this as needed will invite inconsistency (which already exists). One PR I want to do is creating a consistent naming standard for these like `<page-title>-<term/concept/property/header>`.

Reply:
I don't think it's much of a guessing game as to whether or not an anchor exists (you can just look), and I think the strategy of "when needed" is as consistent as it needs to be.

I think the litmus test to use is whether or not taking the time to proactively make everything consistent would make things easier/better to work with. I don't really see any meaningful upside of adding a bunch of anchors that don't do anything - it'd just be a waste of time. (I _am_ ok with trying to standardize naming, though.)